ironfish-cli/src/commands/deposit.ts

```diff
@@ -88,9 +88,15 @@ export default class Bank extends IronfishCommand {
     Assert.isNotUndefined(graffiti)
 
     const balanceResp = await this.client.getAccountBalance({ account: accountName })
-    const newBalance = oreToIron(
-      Number(balanceResp.content.confirmed) - ironToOre(IRON_TO_SEND) - fee,
-    )
+    const confirmedBalance = Number(balanceResp.content.confirmed)
+    if (confirmedBalance < ironToOre(IRON_TO_SEND) + fee) {
```

Comment:
double checking, could this also be `<=`?

Reply:
should be okay. if you need 0.11 and you have 0.11, you can send, youll just end up with a balance of 0

Reply:
:shipit: